docs/index.md

```diff
@@ -0,0 +1,43 @@
+# Getting Started
+
+Generate an HPXML model of home energy use, calibrated to utility data.
+
+!!! note "Nearly everyone will interact solely with the command line interface"
+    Custom workflows can be built using the classes & methods. See the `Code Documentation` section for more information. For advanced users only.
+
+- `openstudio-hpxml-calibration --help`
+- Or the short version: `oshc -h`
+
+## [Required inputs](inputs.md)
+
+- An [HPXML model](https://hpxml.nrel.gov/)
+    - Possibly generated by an [OSW file](inputs.md#osw-file-defining-the-home) (JSON) for the `BuildResidentialHPXML` measure
```

Comment:
Drop this line. It's out of scope for this project and the vast majority of users would not need it. 
```suggestion
```